modules/rtm/src/exp_int.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+double exp_int(double k){
+    double tau;
+    double xx, yy;
+    if(k <= 0.0){
+        tau = 1;
+    }
+    else if (k > 0.0 and k <= 4.0){
+        xx = 0.5 * k - 1.0;
+        yy=(((((((((((((((-3.60311230482612224e-13
+        *xx+3.46348526554087424e-12)*xx-2.99627399604128973e-11)
```

Comment:
These are exactly the coefficients used in previous FORTRAN scripts (i.e. I didn't come up with them). This function is a numerical approximation of the exponential integral, which propagates through the enire calculation of the PROSPECT model. I suppose I could take the time to figure out what the minimum working precision of these values was, but that seems like a waste of time if we already have these.